bundles/integration/admin-layerselector/views/tabPanelView.js

```diff
@@ -217,34 +229,34 @@ define([
              * @param  {Object} model data to populate the form with
              * @return {DOMElement}  element ready to be added to UI
              */
-            __createGroupingPanel : function(tabId, model) {
+            __createGroupingPanel: function(tabId, model) {
                 var instance = this.options.instance,
                     adminLoc = instance.getLocalization('admin'),
                     groupingConfig = {
                         "data": model,
                         // localizations
                         "title": adminLoc.addOrganizationName,
-                        "desc" : adminLoc.addOrganizationNameTitle,
-                        "localeInput" : adminLoc,
-                        "btnLoc" : {
-                            "add" : {
-                                "title" : instance.getLocalization('add'),
-                                "desc" : adminLoc.addNewOrganization
+                        "desc": adminLoc.addOrganizationNameTitle,
+                        "localeInput": adminLoc,
+                        "btnLoc": {
+                            "add": {
+                                "title": instance.getLocalization('add'),
+                                "desc": adminLoc.addNewOrganization
                             },
-                            "save" : {
-                                "title" : instance.getLocalization('save')
+                            "save": {
+                                "title": instance.getLocalization('save')
                             },
-                            "delete" : {
-                                "title" : instance.getLocalization('delete')
+                            "delete": {
+                                "title": instance.getLocalization('delete')
                             },
-                            "cancel" : {
-                                "title" : instance.getLocalization('cancel')
+                            "cancel": {
+                                "title": instance.getLocalization('cancel')
                             }
                         }
-                };
+                    };
 
                 // override some UI texts for inspire theme form
-                if(tabId === 'inspire') {
+                if (tabId === 'inspire') {
```

Comment:
Should this be layergroup instead of inspire?

Reply:
I guess it can be inspire since that's what the localization says in this bundle. So no need to change it.